test/python/test_status_api.py

```diff
@@ -0,0 +1,208 @@
+from __future__ import absolute_import
+
+import unittest
+
+import openapi_client
+
+from . import api_config
+
+WEBSERVICE_POLICY = """
+- !policy
+  id: conjur/authn-oidc/okta
+  body:
+  - !webservice
+    id: status
+    annotations:
+      description: Status service to check that the authenticator is configured correctly
+
```

Comment:
Thinking this might do well in separate file.
There are also a few formatting errors near the bottom, just some missing tabs.